index.html

```diff
@@ -57,15 +57,15 @@
     </div>
 
     <!-- About section -->
-    <div id="about">
+    <section id="about">
       <div>
           E-millenial Store is an hypothetical online store for gadjets we
           beleive every Millenial or Gen-Z should have today. We've got mobile
           phones, laptops, smart watches and other gadgets. Explore all our 
           gadjets and choose the one that suit you.
          
       </div>
-  </div>
+    </section>
 
   <div id="shop">
     <div>
```

Comment:
Use [semantic elements](https://www.w3schools.com/html/html5_semantic_elements.asp) which clearly define its content as opposed to non-semantic elements that tell nothing about its contents. Examples of semantic elements are:

- form
- main
- article
and examples of non-semantics are:

- div
- span

Reply:
Good job @flemton 🙏